front/pages/api/v1/w/[wId]/spaces/[spaceId]/data_sources/[dsId]/tables/index.ts

```diff
@@ -276,6 +277,18 @@ async function handler(
         source_url: sourceUrl,
       } = r.data;
 
+      // Labs: Salesforce personal connections, don't upsert secretId
```

Comment:
This will prevent use in the API but you mentioned table query blocks in dust apps as well, how do we prevent use of the admin token there?

Reply:
This prevents also dust apps, as the connectionId is just not available at all in core. Only table_queries is passing the connectionId in the dust_app config - if you don't pass it in config, and if it's not available in the table itself, core throws a require_authentication error.